docs/Contributing/ContributingDocumentation.mdr

```diff
@@ -5,19 +5,26 @@
 You might be here because you found something you could improve in [Gwion's documentation](https://github.com/fennecdjay/gwion-docs).
 Please open an [issue](https://github.com/fennecdjay/gwion-docs/issues) describing the problem and how you might address it.
 
-
 ## Dependencies
 
-  * Gwion
-  * mdr
-  * MdBook
+Technically none of these are *needed*, but they are quite useful:
+
+* Gwion
+* mdr
+* MdBook
```

Comment:
I think this misses links.
 * gwion:  https://github.com/fennecdjay/gwion
 * mdr:    https://github.com/fennecdjay/mdr
 * mdbook: https://github.com/rust-lang/mdBook

Reply:
Okay, Ill get that in.

Reply:
Ok, done.